frontends/web/src/routes/device/bitbox02/backups.tsx

```diff
@@ -72,18 +72,16 @@ class Backups extends Component<Props, State> {
     if (this.props.backupOnBeforeRestore) {
       this.props.backupOnBeforeRestore(backup);
     }
-    apiPost(
-      'devices/bitbox02/' + this.props.deviceID + '/backups/restore',
-      this.state.selectedBackup).then(({ success }) => {
-      this.setState({
-        restoring: false,
-        errorText: success ? '' : this.props.t('backup.restore.error.general'),
+    restoreBackup(this.props.deviceID, this.state.selectedBackup)
+      .then(({ success }) => {
+        this.setState({
+          restoring: false,
+          errorText: success ? '' : this.props.t('backup.restore.error.general'),
+        });
+        if (this.props.backupOnAfterRestore) {
+          this.props.backupOnAfterRestore(success);
```

Comment:
this works even if success is `undefined` instead of false, but technically the type is wrong as it expects boolean.

https://github.com/digitalbitbox/bitbox-wallet-app/blob/82b3291732d0e5030de43f1217caeadb2d84f91b/frontends/web/src/routes/device/bitbox02/bitbox02.tsx#L315-L320

Reply:
will be fixed by https://github.com/digitalbitbox/bitbox-wallet-app/pull/1933